velox/expression/tests/CastExprTest.cpp

```diff
@@ -173,6 +173,159 @@ class CastExprTest : public functions::test::CastBaseTest {
     assertEqualVectors(expected, result);
   }
 
+  template <typename T>
+  void testDecimalToFloatCasts(TypePtr type) {
```

Comment:
Is the `type` argument used anywhere?

Reply:
This is left over from previous reviews. Previously, testDecimalToIntegralCastsOutOfBounds test was inside testDecimalToIntegralCasts, hence there was need of type. No longer. Fixed.